sentry-samples/sentry-samples-android/src/main/AndroidManifest.xml

```diff
@@ -23,6 +23,7 @@
         android:theme="@style/AppTheme"
         android:allowNativeHeapPointerTagging="false"
         android:extractNativeLibs="true"
+        android:forceQueryable="true"
```

Comment:
related to https://github.com/android/android-test/issues/743 ? lets add a comment if so

Reply:
This was one of approaches i tested, not needed anymore.